File: Eventili/src/entities/imagepers.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package entities;

/**
 *
 * @author msi
 */
public class imagepers {
       private int id_imp,id_pers;
    private String imageP,last;

    public imagepers(){
        
    }
    public imagepers(String imageP,String last,int id_pers) {
        this.id_pers = id_pers;
        this.imageP = imageP;
        this.last=last;
    }
    public imagepers(int id_imp, String imageP,String last, int id_pers) {
        this.id_imp = id_imp;
        this.id_pers = id_pers;
        this.imageP = imageP;
    }

    @Override
    public String toString() {
        return "imagepers{" + "id_imp=" + id_imp + ", id_pers=" + id_pers + ", imageP=" + imageP + ", last=" + last +'}';
    }

    public String getLast() {
        return last;
    }

    public void setLast(String last) {
        this.last = last;
    }

    public int getId_imp() {
        return id_imp;
    }

    public void setId_imp(int id_imp) {
        this.id_imp = id_imp;
    }

    public int getId_pers() {
        return id_pers;
    }

    public void setId_pers(int id_pers) {
        this.id_pers = id_pers;
    }

    public String getImageP() {
        return imageP;
    }

    public void setImageP(String imageP) {
        this.imageP = imageP;
    }
    
}

File: web/eventiliProject/src/Entity/Imagepers.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;
use App\Repository\ImagePersRepository;


#[ORM\Entity(repositoryClass: ImagepersRepository::class)]

class Imagepers
{
    #[ORM\Id]
    #[ORM\GeneratedValue]
    #[ORM\Column]
    private ?int $idImp=null;
    
    
    #[ORM\Column(length:255)]
    private ?String $imagep=null;
   
    #[ORM\Column(length:255)]
    private ?String $last=null;
   
    #[ORM\ManyToOne(targetEntity: Personne::class)]
    #[ORM\JoinColumn(name: "id_pers", referencedColumnName: "id_pers")]
    private ?Personne $idPers=null;
    
    /*#[ORM\ManyToOne(inversedBy: 'Personne')]
    private ?Personne $idPers=null;*/

    public function getIdImp(): ?int
    {
        return $this->idImp;
    }

    public function getImagep(): ?string
    {
        return $this->imagep;
    }

    public function setImagep(string $imagep): self
    {
        $this->imagep = $imagep;

        return $this;
    }

    public function getLast(): ?string
    {
        return $this->last;
    }

    public function setLast(string $last): self
    {
        $this->last = $last;

        return $this;
    }

    public function getIdPers(): ?Personne
    {
        return $this->idPers;
    }

    public function setIdPers(?Personne $idPers): self
    {
        $this->idPers = $idPers;

        return $this;
    }


}
